src/CachePlugin.php

```diff
@@ -353,13 +358,15 @@ private function configureOptions(OptionsResolver $resolver)
             'respect_response_cache_directives' => ['no-cache', 'private', 'max-age', 'no-store'],
             'cache_key_generator' => null,
             'cache_listeners' => [],
+            'blacklisted_paths' => [],  // restricted for
```

Comment:
please remove the comment, the phpdoc on the constructor should document this (plus the documentation change)

Reply:
done